ament_cmake_test/ament_cmake_test/__init__.py

```diff
@@ -216,7 +216,7 @@ def log(msg, **kwargs):
             print(decoded_line, end='')
             output += decoded_line
             if output_handle:
-                output_handle.write(decoded_line.encode())
+                output_handle.write(decoded_line.encode(errors='surrogateescape'))
```

Comment:
@ejalaa12 maybe be simpler to use `'replace'` rather than `'surrogateescape'`

Reply:
Completely agree.

Reply:
@hidmic  should I also use `'replace'` for the decoding ?